src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static StringBuilder builder;
+    private static final DateTimeFormatter DOT_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int NAME_INDEX = 1;
+    private static final int DATE_INDEX = 0;
+    private static final int SALARY_INDEX = 3;
+    private static final int HOUR_INDEX = 2;
+    private static LocalDate date;
+
+    private String[] splited;
+    private int salarySum;
```

Comment:
let's make all non-constant variables local (declare them in method, and not in class)